Makefile.envs

```diff
@@ -2,6 +2,7 @@ export PYVERSION ?= 3.12.1
 export PYODIDE_EMSCRIPTEN_VERSION ?= 3.1.58
 export PYODIDE_VERSION ?= 0.27.0.dev0
 export PYODIDE_ABI_VERSION ?= 2024_0
+export PYODIDE_BUILD_VERSION ?= 0.27.3
```

Comment:
I wonder if we should add optional `PYODIDE_BUILD_REPO` and `PYODIDE_BUILD_COMMIT` -- if these are defined, we can check in the `pyodide_build` rule that `direct_url.json` exists and has appropriate contents. Then perhaps we should install `pyodide_build` everywhere with `make pyodide_build`...?

Reply:
Sounds reasonable to me. Let me see if I can make if work.